adapters/handlers/rest/handlers_schema.go

```diff
@@ -286,7 +288,38 @@ func (s *schemaHandlers) getTenants(params schema.TenantsGetParams,
 	}
 
 	s.metricRequestsTotal.logOk(params.ClassName)
-	return schema.NewTenantsGetOK().WithPayload(tenants)
+	return schema.NewTenantsGetOK().WithPayload(schemaUC.TenantResponsesToTenants(tenants))
+}
+
+func (s *schemaHandlers) getTenant(
+	params schema.TenantsGetOneParams,
+	principal *models.Principal,
+) middleware.Responder {
+	tenants, err := s.manager.GetConsistentTenants(params.HTTPRequest.Context(), principal, params.ClassName, *params.Consistency, []string{params.TenantName})
```

Comment:
**thought:**  do you think it will need authZ or this for internal usage ? 

i think it shall have `authZ` checks given that it's exposed to the open API

e.g.
```
h.authorizer.Authorize(principal, authorization.READ, authorization.Shards(params.ID)...)
```

Reply:
Currently, that auth check happens inside the `s.manager.GetConsistentTenants` method call (same as in the existing `getTenants` method above). I assume that's to share the auth call among both the HTTP and gRPC handlers.

But separate from this PR I'm curious how we should ensure that the auth checks are not missed on any endpoints...

Reply:
Currently, that auth check happens inside the s.manager.GetConsistentTenants method call (same as in the existing getTenants method above). I assume that's to share the auth call among both the HTTP and gRPC handlers.

But separate from this PR I'm curious how we should ensure that the auth checks are not missed on any endpoints...